cmd/kwil-cli/cmds/database/call.go

```diff
@@ -18,33 +18,32 @@ import (
 )
 
 var (
-	callLong = `Call a ` + "`" + `view` + "`" + ` action, returning the result.
+	callLong = `Call a ` + "`" + `view` + "`" + ` procedure or action, returning the result.
 
-` + "`" + `view` + "`" + ` actions are read-only actions that do not require gas to execute.  They are
+` + "`" + `view` + "`" + ` procedure are read-only procedure that do not require gas to execute.  They are
 the primary way to query the state of a database. The ` + "`" + `call` + "`" + ` command is used to call
-a ` + "`" + `view` + "`" + ` action on a database.  It takes the action name as a required flag, and the
-action inputs as arguments.
+a ` + "`" + `view` + "`" + ` procedure on a database.  It takes the procedure name as the first positional
+argument, and the procedure inputs as all subsequent arguments.
 
-To specify an action input, you first need to specify the input name, then the input value, delimited by a colon.
-For example, for action ` + "`" + `get_user($username)` + "`" + `, you would specify the action as follows:
+To specify a procedure input, you first need to specify the input name, then the input value, delimited by a colon.
+For example, for procedure ` + "`" + `get_user($username)` + "`" + `, you would specify the procedure as follows:
 
-` + "`" + `username:satoshi` + "`" + ` --action=get_user
+` + "`" + `call get_user username:satoshi` + "`" + `
 
 You can either specify the database to execute this against with the ` + "`" + `--name` + "`" + ` and ` + "`" + `--owner` + "`" + `
 flags, or you can specify the database by passing the database id with the ` + "`" + `--dbid` + "`" + ` flag.  If a ` + "`" + `--name` + "`" + `
 flag is passed and no ` + "`" + `--owner` + "`" + ` flag is passed, the owner will be inferred from your configured wallet.
 
 If you are interacting with a Kwil gateway, you can also pass the ` + "`" + `--authenticate` + "`" + ` flag to authenticate the call with your private key.`
 
-	callExample = `# Calling the ` + "`" + `get_user($username)` + "`" + ` action on the "mydb" database
-kwil-cli database call --action get_user --name mydb --owner 0x9228624C3185FCBcf24c1c9dB76D8Bef5f5DAd64 username:satoshi
+	callExample = `# Calling the ` + "`" + `get_user($username)` + "`" + ` procedure on the "mydb" database
```

Comment:
NBD, maybe `get_user($username) view`?

Reply:
Eh, I think it is less clear

Reply:
It's calling a view action/procedure, but nbd.